app/src/main/res/layout-sw320dp/activity_description.xml

```diff
@@ -73,14 +74,14 @@
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
                                 android:fontFamily="@font/armata"
-                                android:text="By " />
+                                android:text="@string/activity_description_by" />
 
                             <TextView
                                 android:id="@+id/activity_organisator_description"
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
                                 android:fontFamily="@font/armata"
-                                android:text="Organisator" />
+                                android:text="@string/activity_description_Organisator" />
                         </LinearLayout>
 
                         <de.hdodenhof.circleimageview.CircleImageView
```

Comment:
I think there is an error here : "Cannot resolve class de.hdodenhof.circleimageview.CircleImageView"

Reply:
It was just for aesthetic purposes but if it makes a mistake I'll just put an imageView back.